evergreen.py

```diff
@@ -282,7 +293,7 @@ def commit_changes(title, body, repo, dependabot_file, message):
     branch_name = "dependabot-" + str(uuid.uuid4())
     repo.create_ref(front_matter + branch_name, default_branch_commit)
     repo.create_file(
-        path=".github/dependabot.yaml",
+        path=".github/" + dependabot_filename,
```

Comment:
found a bug here where it adds ".github/ twice. fixing locally and will push new commit to fix.